packages/apps/plugins/plugin-status-bar/src/components/StatusBar.tsx

```diff
@@ -66,7 +69,9 @@ const StatusBarContainer = forwardRef<HTMLDivElement, StatusBarContainerProps>(
         {...groupAttrs}
         className={mx(
           'bs-[--statusbar-size]',
-          'flex justify-end items-center gap-2 border-solid border-t border-gray-200 bg-gray-50 text-gray-500',
+          'flex justify-end items-center gap-2',
+          'surface-base fg-description',
+          'border-solid border-t border-bs separator-separator',
```

Comment:
```suggestion
          'border-bs separator-separator',
```
`border-solid` is inferred by `border-bs`, and `border-t` is overridden by `border-bs`